recipe/meta.yaml

```diff
@@ -14,9 +14,9 @@ source:
     - patches/0003-Don-t-enable-debug-info-for-all-builds.patch
 
 build:
-  number: 0
+  number: 1
   osx_is_app: True  # [osx]
-  skip: True  # [win and vc<14]
+  skip: True  # [win and vc<14 or osx and py<37]
```

Comment:
Just to be on the safe side can you encapsulate these in parentheses?

`[(win and vc<14) or (osx and py<37)]`